test/stripe/api_resource_test.rb

```diff
@@ -429,7 +491,21 @@ class NestedTestAPIResource < APIResource
         ch.save
       end
 
-      should "correctly handle replaced nested objects" do
+      should "not update nested API resources" do
+        Stripe::Charge.construct_from(id: "ch_id",
+                                      customer: {
+                                        object: "customer",
+                                        id: "customer_id",
+                                      })
+
+        stub_request(:post, "#{Stripe.api_base}/v1/charges/ch_id")
+          .with(body: {})
+          .to_return(body: JSON.generate("id" => "ch_id"))
+
+        Stripe::Charge.update("ch_id", { customer: { description: "Bob" } })
+      end
+
```

Comment:
I don't get this one either.
```suggestion
      should "not update nested API resources" do
        Stripe::Charge.construct_from(id: "ch_id",
                                      customer: {
                                        object: "customer",
                                        id: "customer_id",
                                      })

        stub_request(:post, "#{Stripe.api_base}/v1/charges/ch_id")
          .with(body: {})
          .to_return(body: JSON.generate("id" => "ch_id"))

        Stripe::Charge.update("ch_id", { customer: { description: "Bob" } })
      end

```